packages/keycloak-service/src/service/serviceClass.ts

```diff
@@ -74,17 +75,18 @@ export const customFetch: CustomFetch = async (...rest) => {
   try {
     return await fetch(...rest);
   } catch (err) {
-    throwNetworkError(err);
+    if (err instanceof Error) throwNetworkError(err);
```

Comment:
[Ts 4.0 now types catch errors as unknown (rather than any)](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0-beta/#unknown-on-catch). In hind sight I think type casting the error should have been better rather than ignoring when not instance of Error